rtsp.go

```diff
@@ -693,11 +693,16 @@ func NewRTSPCamera(ctx context.Context, _ resource.Dependencies, conf resource.C
 
 	framePool := newFramePool(initialFramePoolSize, logger)
 
+	rtpPassthrough := true
+	if newConf.RTPPassthrough != nil {
+		rtpPassthrough = *newConf.RTPPassthrough
+	}
+
```

Comment:
Isn't this constructor shared by the other models that do not support rtp passthrough as well? mjpeg, h265?

Reply:
Yeah it is I think it's handled on a lower level when it bypasses passthrough after it checks that the properties of the stream isn't h264